src/cheri_riscv_types.sail

```diff
@@ -24,27 +24,46 @@ function ext_accum_ptw_result(ptw : ext_ptw, pte : pte_check) -> ext_ptw =
   }
 
 /* Address translation errors */
-enum ext_ptw_error = {AT_LOAD_CAP_ERR, AT_STORE_CAP_ERR}
+enum ext_ptw_error = {AT_CAP_ERR}
 
 /* CHERI exception extensions */
 
-enum ext_exc_type = {EXC_CHERI, EXC_CHERI_VMEM_LOAD_CAP, EXC_CHERI_VMEM_STORE_CAP}
+enum ext_exc_type = {
+  EXC_LOAD_CAP_PAGE_FAULT,
+  EXC_SAMO_CAP_PAGE_FAULT,
+  EXC_CHERI
+}
 
 /* CHERI translation of PTW errors into exception annotations */
 function ext_translate_exception(e : ext_ptw_error) -> ext_exc_type =
   match e {
-    AT_LOAD_CAP_ERR  => EXC_CHERI_VMEM_LOAD_CAP,
-    AT_STORE_CAP_ERR => EXC_CHERI_VMEM_STORE_CAP
-}
+    AT_LOAD_CAP_ERR  => EXC_LOAD_CAP_PAGE_FAULT,
+    AT_STORE_CAP_ERR => EXC_SAMO_CAP_PAGE_FAULT
+  }
```

Comment:
This hunk seems stale -- I think `ext_translate_exception` is not used any more as of this commit and should just be removed.  (It's only called from within `riscv_ptw.sail` upstream, a file we override).

Reply:
I view it as a bug that we have to do so. But yes, this hook doesn't make sense in its current form and should just go.